frontend/src/main/components/Jobs/UpdateCowHealthJobForm.js

```diff
@@ -0,0 +1,60 @@
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+function UpdateCowHealthJobForm({ submitAction }) {
+
+ const defaultValues = {
+    fail: false,
+    sleepMs: 1000
```

Comment:
Same as abov... we don't need parameters.  An empty `defaultValues` is ok though.